.github/actions/install-dependencies/action.yml

```diff
@@ -25,6 +25,6 @@ runs:
       if: steps.cache.outputs.cache-hit != 'true'
 
     - name: Run postinstall script
-      run: yarn postinstall && yarn workspace @razorpay/blade postinstall
+      run: yarn postinstall
```

Comment:
Why this change?

Reply:
postinstall script is no longer required, it was just running patch-package for bundlemon & we no longer depend on bundlemon.